types/puppeteer/index.d.ts

```diff
@@ -1073,6 +1074,19 @@ export interface RemoteInfo {
     port: number;
 }
 
+export interface SecurityDetails {
+    /** Returns a string with the name of issuer of the certificate. (e.g. "Let's Encrypt Authority X3"). */
+    issuer(): string;
+    /** Returns the cryptographic protocol (e.g. TLS 1.2). */
+    protocol(): string;
+    /** Returns the name of the subject to which the certificate was issued to (e.g. "www.example.com"). */
+    subjectName(): string;
+    /** Returns the unix timestamp of when the certificate was registered. */
```

Comment:
NIT: This is not strictly correct, registration != start of validity, you might want to copy/paste the wording from the actual docs.

Reply:
@SimonSchick I agree, I can make this change in the morning (I'm on EET timezone)

Reply:
@SimonSchick done